mobify/platform-scaffold
# WEB-1335 Make Merlin's footer more app-like

(brief description / PR title)

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1335

## Changes
- Changed newsletter heading to h4
- changed button to inline tertiary style
- added additional fieldrow wrapper to align inline
- removed List component from links, changed to Link component
- changed link styling
- Moved social icons to navigation menu
- Moved copyright to navigation menu

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- scroll to footer
- open menu to check content in navigation
